test/conformance/parallel/configuration.go

```diff
@@ -0,0 +1,73 @@
+package test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+type yamlTimeDur time.Duration
+
+type MasterOffsetContinuous struct {
```

Comment:
why it is name as MasterOffsetContinuous, `offset` is one of the  many testcase right ?

Reply:
example  FAULT, offset staying above  MAXOFFSET for 5 secs or n secs etc .Process Restart count etc